bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller.go

```diff
@@ -819,6 +854,27 @@ func (r *KubeadmConfigReconciler) resolveSecretPasswordContent(ctx context.Conte
 	return data, nil
 }
 
+// skipTokenRefreshIfExpiringAfter returns a duration from "now". If the token's expiry timestamp is after
+// `now + skipTokenRefreshIfExpiringAfter()`, it does not yet need a refresh.
+func (r *KubeadmConfigReconciler) skipTokenRefreshIfExpiringAfter() time.Duration {
```

Comment:
Was the name/comment a leftover? The function just returns a duration, rather than a duration from `now`